thirdparty/openldap/zimbra-openldap/debian/control

```diff
@@ -12,7 +12,7 @@ Package: zimbra-openldap-server
 Priority: optional
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, zimbra-openldap-lib (= ${binary:Version}),
- zimbra-ldap-base
+ zimbra-ldap-base, zimbra-libsodium-dev (>= 1.0.19-1zimbra10.0b1ZAPPEND)
```

Comment:
You probably want `zimbra-libsodium-lib` here, instead of `-dev`.

Reply:
right, updated it.